torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13164,6 +13164,17 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            skips=(
                DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),),
            ),
+    OpInfo('expand_copy',
+           # op=lambda self, shape: self.expand(shape),
```

Comment:
Is this stale from a copy-paste?

Reply:
Fixed! Still waiting for very slow CI to commit this.

Hope you're having fun out there!!